avocado/utils/lv_utils.py

```diff
@@ -54,6 +56,38 @@ def get_diskspace(disk):
     raise LVException('Error in finding disk space')
 
 
+def get_device_total_space(device):
+    """Get the total device size.
+
+    :param str device: name of the device/disk to find the total size
+    :returns: size in bytes
+    :rtype: str
```

Comment:
This method return an `int`, right?

Reply:
This maintains the previous functionality, so it is returning `str` like before.